client/scripts/bundle.js

```diff
@@ -200,16 +200,16 @@ async function main() {
         "@nomicfoundation/solidity-analyzer-linux-x64-musl": "0.1.1",
         "@nomicfoundation/solidity-analyzer-freebsd-x64": "0.1.1",
 
-        "@nomicfoundation/slang": "0.10.1",
-        "@nomicfoundation/slang-darwin-arm64": "0.10.1",
-        "@nomicfoundation/slang-win32-arm64-msvc": "0.10.1",
-        "@nomicfoundation/slang-linux-arm64-gnu": "0.10.1",
-        "@nomicfoundation/slang-linux-arm64-musl": "0.10.1",
-        "@nomicfoundation/slang-win32-ia32-msvc": "0.10.1",
-        "@nomicfoundation/slang-darwin-x64": "0.10.1",
-        "@nomicfoundation/slang-win32-x64-msvc": "0.10.1",
-        "@nomicfoundation/slang-linux-x64-gnu": "0.10.1",
-        "@nomicfoundation/slang-linux-x64-musl": "0.10.1",
+        "@nomicfoundation/slang": "0.13.1",
```

Comment:
nit: Is it worth it to start adding sync markers in such places, since you already did the effort of identifying all of them?

```suggestion
        // __SLANG_NPM_PACKAGE_VERSION__ (keep in sync)
        "@nomicfoundation/slang": "0.13.1",
```